sdgx/metrics/pair_column/mi_sim.py

```diff
@@ -0,0 +1,120 @@
+import time
+from datetime import datetime
+
+import numpy as np
+import pandas as pd
+from scipy.stats import entropy
+from sklearn.metrics.cluster import normalized_mutual_info_score
+
+from sdgx.metrics.pair_column.base import PairMetric
+
+
+def Jaccard_index(A, B):
+    return min(A, B) / max(A, B)
+
+
+def time2int(datetime, form):
+    time_array = time.strptime(datetime, form)
+    time_stamp = int(time.mktime(time_array))
+    return time_stamp
```

Comment:
can we add this method to our project such as `sdgx.utils.py`?  or integrate this functiona into `formatters`

Reply:
Good idea

Reply:
we may deal with this kind of funcs later when implementing `formatters`